Changelogs for Digital Twin Manager Microservice 1024.0.0 (#4135)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-0-0-fixed-slow-performance-when-creating-linked-series-without-a-source-on-tenants-with-lots-of-manged-objects.md

---
date: ""

> title: "Fixed slow performance when creating linked series without a source on tenants with lots of MangedObjects."

Reviewer: ```suggestion
title: "Fixed slow performance when creating a linked series without a source on tenants with lots of MangedObjects."
```

> title: "Fixed slow performance when creating a linked series without a source on tenants with lots of MangedObjects."

Reviewer 2: ```suggestion
title: "Fixed slow performance when creating a linked series without a source on tenants with lots of managed objects"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2678"
version: "1024.0.0"
---

> Fixed slow performance when creating linked series without a source on tenants with lots of MangedObjects.

Reviewer: ```suggestion
Previously on tenants with a high number of MangedObjects, creating a linked series without a source caused a slow performance due to the unnecessary loading of inventory objects. This is fixed now.
```

> Previously on tenants with a high number of MangedObjects, creating a linked series without a source caused a slow performance due to the unnecessary loading of inventory objects. This is fixed now.

Reviewer 2: ```suggestion
On tenants with a high number of managed objects, creating a linked series without a source caused a slow performance due to the unnecessary loading of inventory objects. This issue has been fixed.
```
